.github/workflows/build_and_deploy.yml

```diff
@@ -374,6 +375,22 @@ jobs:
         if: ${{ !matrix.settings.docker }}
         shell: bash
 
+      # Trying to upload metrics for the Turbopack to datadog's CI pipeline execution
+      - name: 'Upload turbopack build metrics'
+        shell: bash
+        continue-on-error: true
+        run: |
+          npm install -g @datadog/datadog-ci
+          for filename in packages/next-swc/native/next-swc.*.node; do
+            # Strip out filename to extract target triple
+            export FILENAME=$(basename ${filename})
+            export FILENAME=${FILENAME#*.}
+            export FILENAME=${FILENAME%.node}
+            export BYTESIZE=$(wc -c < $filename | xargs)
```

Comment:
maybe? somehow when I try to make pass I added it to make work. Can we leave it as-is for now since it is working? It was quite painful to verify this waiting CI to complete all of the native binaries, and would like to refactor / optimize later.